src/platform_impl/macos/util/async.rs

```diff
@@ -88,6 +88,13 @@ pub unsafe fn set_level_async(ns_window: id, level: ffi::NSWindowLevel) {
     });
 }
 
+pub unsafe fn set_window_ignore_cursor_events(ns_window: id, ignore: bool) {
```

Comment:
I believe this should be called `set_ignore_cursor_events_async`. The "window" is removed because it doesn't add any information and the "async" is added telling the caller that this change will not have taken effect by the time the function returns.